<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordInserter</class>
 <widget class="QMainWindow" name="WordInserter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word Inserter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="inputFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="inputLabel">
          <property name="text">
           <string>Text input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="inputPlainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="outputFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="outputLabel">
          <property name="text">
           <string>Text output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="outputTextBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="actionButton">
        <property name="text">
         <string>Transform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
